scripts/configure.py

```diff
@@ -157,17 +157,17 @@
     ]
   },
   'lms' : {
-    'apt': ['libcrypt-openssl-rsa-perl', 'libio-socket-ssl-perl'], # needed for ShairTunes2W support
+    'apt': ['libcrypt-openssl-rsa-perl', 'libio-socket-ssl-perl', 'libopusfile0'],
```

Comment:
`squeezelite` has this dep but we're installing using `dpkg`, which does not do dependencies. #472 would obviate this need (and frankly save a lot of disk IO, upgrade time, compiling time elsewhere, etc...)